mappings/net/minecraft/client/gui/Gui.mapping

```diff
@@ -22,6 +22,7 @@ CLASS none/bgn net/minecraft/client/gui/Gui
 		ARG 0 mouseX
 		ARG 1 mouseY
 		ARG 2 mouseButton
+	METHOD a mouseDragged (IIIJ)V
```

Comment:
I'm not against `onMouseDragged` but the Mojang name is `mouseDragged` (from Realms).